modules/swagger-parser-v2-converter/src/main/java/io/swagger/v3/parser/converter/SwaggerConverter.java

```diff
@@ -561,6 +562,7 @@ public Operation convert(io.swagger.models.Operation v2Operation) {
                     formParams.add(param);
                 } else if ("body".equals(param.getIn())) {
                     operation.setRequestBody(convertParameterToRequestBody(param, v2Operation.getConsumes()));
+                    operation.addExtension("x-codegen-request-body-name", param.getName());
```

Comment:
It seems as though the `name` field's intent has been replaced by `title`, as evidenced by how the string `"body"` gets resolved in the converter: https://github.com/swagger-api/swagger-parser/blob/670b278f1cf925ce93a8d3c17a1c8411d9e5329b/modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/util/InlineModelResolver.java#L67

I would propose a different direction, instead of adding a vendor extension, to just `setTitle(param.getName())` and let the rest of the infrastructure wire the parameter through.

This will seamlessly resolve the issue for those who have migrated to the new `title` infrastructure.

Reply:
@gracekarina Out of curiosity, any thoughts on the direction of this comment? I still suspect adding a vendor extension that duplicates in-built behavior is causing a needless fork in functionality, but maybe I've misunderstood an aspect about the semantics between `name` and `title` here.

Reply:
Hi @blast-hardcheese I'll review the comment you made, if what you say is right I'll revert the PR. thanks for pointing us to that.